### PR TITLE
Minors fixes for MongoDB and Python query runners

### PR DESCRIPTION
**Pull request by erans**
- Python: added 'additionalModulesPaths' config key to add additional paths (comma separated) from which an import can occur. It still requires white listing the module in 'allowedImportModules'

- MongoDB - Added support for bson.timestamp.Timestamp JSON serialization as it sometimes gets returned in newer PyMongo versions.

### CODE REVIEW

File: redash/query_runner/python.py [hunk 1]
```diff
@@ -144,7 +153,9 @@ def execute_query(self, data_source_name_or_id, query):
         except models.DataSource.DoesNotExist:
             raise Exception("Wrong data source name/id: %s." % data_source_name_or_id)
 
-        data, error = data_source.query_runner.run_query(query)
+        query_runner = get_query_runner(data_source.type, data_source.options)
```
Comment (arikfr) [inline redash/query_runner/python.py]: get_query_runner doesn't exist anymore.

Comment (erans) [inline redash/query_runner/python.py]: Really? When did it go? Is it in a specific branch?

I'll fix it.

**Comment by erans** [comment]
Updated the code

File: redash/query_runner/python.py (hunk 1 above)
Comment (erans) [inline redash/query_runner/python.py]: I've updated the code.

File: redash/query_runner/python.py [hunk 2]
```diff
@@ -56,8 +60,8 @@ def enabled(cls):
     def annotate_query(cls):
         return False
 
-    def __init__(self, configuration):
-        super(Python, self).__init__(configuration)
+    def __init__(self, configuration_json):
+        super(Python, self).__init__(configuration_json)
```
Comment (arikfr) [inline redash/query_runner/python.py]: You reverted a change (it should be `configuration`).

**Pull request merged**